src/web/src/components/Footer.vue

```diff
@@ -46,21 +49,42 @@
 
 import { getSemesters } from '@/services/YacsService';
 
+import { getSemester }  from '@/services/AdminService';
```

Comment:
can we rename this to getDefaultSemester as that is what it represents, getSemester is ambiguous

Reply:
Done